descriptors/ModuleDescriptor-template.json

```diff
@@ -231,6 +231,14 @@
             "orders-storage.sources.item.delete",
             "orders-storage.vendor_details.item.delete"
           ]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/orders/po-number/validate",
+          "permissionsRequired": ["orders.po-number.item.post"],
+          "modulePermissions": [
+            "orders-storage.purchase_orders.item.get"
```

Comment:
Please change this permission to `orders-storage.purchase_orders.collection.get`

Reply:
Good find. Thank you